cpp/src/codegen/llvm_generator_test.cc

```diff
@@ -33,6 +33,21 @@ class TestLLVMGenerator : public ::testing::Test {
   FunctionRegistry registry_;
 };
 
+// Verify that a valid pc function exists for every function in the registry.
+TEST_F(TestLLVMGenerator, VerifyPCFunctions) {
+  std::unique_ptr<LLVMGenerator> generator;
+  Status status = LLVMGenerator::Make(ConfigurationBuilder::DefaultConfiguration(),
+                                      &generator);
+  EXPECT_TRUE(status.ok());
+
+  llvm::Module *module = generator->module();
+  for (auto iter = registry_.begin(); iter != registry_.end(); ++iter) {
```

Comment:
i think we are using for each across the board..

Reply:
will fix